tests/e2e/specs/add-paid-campaigns/add-paid-campaigns.test.js

```diff
@@ -215,7 +215,7 @@ test.describe( 'Set up Ads account', () => {
 
 			await setupAdsAccounts.mockAdsStatusClaimed();
 
-			await page.reload();
+			await page.waitForTimeout( 30000 );
```

Comment:
How about using another trick to speed up this test case?

```suggestion
			await page.dispatchEvent( 'body', 'blur' );
			await page.dispatchEvent( 'body', 'focus' );
```